falco/ci/ci-values.yaml

```diff
@@ -2,9 +2,10 @@
 # The following values will bypass the installation of the kernel module
 # and disable the kernel space driver.
 
-# disable the kernel space driver
+# disable the driver-loader
 driver:
-  enabled: false
+  loader:
+    enabled: false
```

Comment:
This can't work. We still need both syscalls enabled and the driver disabled to make the `--userspace` workaround work.

As suggested in my other comment, a helper could help.